client/src/main/java/com/aliyun/emr/rss/client/ShuffleClient.java

```diff
@@ -193,7 +194,7 @@ public abstract RssInputStream readPartition(
       String applicationId, int shuffleId, int partitionId, int attemptNumber) throws IOException;
 
   /**
-   * 注销
+   * unregisteration
    *
    * @param applicationId
```

Comment:
How about remove this line of `param`? Since it's so clear. cc @waitinfuture @FMX @pan3793

Reply:
Agree

Reply:
+1, let's remove useless Javadoc or supply clear description